guacamole/src/main/frontend/src/translations/en.json

```diff
@@ -826,6 +842,7 @@
         "FIELD_HEADER_COLOR_SCHEME"   : "Color scheme:",
         "FIELD_HEADER_CREATE_RECORDING_PATH" : "Automatically create recording path:",
         "FIELD_HEADER_CREATE_TYPESCRIPT_PATH" : "Automatically create typescript path:",
+        "FIELD_HEADER_CLIPBOARD_BUFFER_SIZE" : "Clipboard data size limit:",
```

Comment:
Please maintain established alphabetical ordering of the translation keys (same for other protocols).

Reply:
Fixed (?).